src/ApprenticeAan/SFA.DAS.ApprenticeAan.Api.UnitTests/Services/QueryStringParameterBuilderTests.cs

```diff
@@ -0,0 +1,227 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using SFA.DAS.ApprenticeAan.Api.Models;
+using SFA.DAS.ApprenticeAan.Application.CalendarEvents.Queries.GetCalendarEvents;
+using SFA.DAS.ApprenticeAan.Application.Common;
+using SFA.DAS.ApprenticeAan.Application.Services;
+
+namespace SFA.DAS.ApprenticeAan.Api.UnitTests.Services;
+
+public class QueryStringParameterBuilderTests
```

Comment:
The target is in Api\Models folder, so this test file needs to be moved to similar folder in the unit test project and it needs to reflect the target name in the filename as well.

Reply:
ok